README.txt

```diff
@@ -10,11 +10,11 @@ at https://github.com/z390development/z390/releases
 This distribution has been regression tested on Windows Server and Ubuntu Linux 
 using GitHub Actions.
 
-Java runtime/SDK used: 
-Eclipse Adoptium OpenJDK 8 https://adoptium.net/?variant=openjdk8
+Java runtime/SDK used for build: 
+Eclipse Adoptium OpenJDK 21 https://adoptium.net/?variant=openjdk21
```

Comment:
build.yml line 7 reads JAVA_DISTRIBUTION: temurin

should adoption be changed to temurin?

Reply:
Official name is Adoptium Temurin - see https://adoptium.net/temurin/releases/?variant=temurin21. 
Will update to use new name.